src/ContainerHeader.jsx

```diff
@@ -8,10 +8,8 @@ class ContainerHeader extends React.Component {
         super(props);
```

Comment:
Why the second commit has `Closes #475`?
And I think the `Fixes` line would be better for the commit that actually touches pods.